model-engine/model_engine_server/db/scripts/fivetran_setup/launch.sql

```diff
@@ -0,0 +1,22 @@
+CREATE USER fivetran PASSWORD 'fivetran';
```

Comment:
are these new and do we want these here?

Reply:
deleting